### PR TITLE
footer added

### PR DESCRIPTION
**Pull request by vedikaag99**
### Brief description of what is fixed or changed
A footer is added for the website. 

## Types of changes

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
Screen Shot 2021-04-21 at 10 24 57 PM

## Other information

This PR is under GSSoC.

### CODE REVIEW

**Comment by ArunTeltia** [comment]
Is this responsive as well ?

**Comment by vedikaag99** [comment]
> Is this responsive as well ?

yes. @ArunTeltia

https://user-images.githubusercontent.com/67699840/115602340-739c1800-a2fc-11eb-9f12-d4b24e3aa667.mov

File: src/components/Footer/footer.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+const Footer = () => {
+  return (
+    <React.Fragment>
+      <footer className={style["footer"]}>
+        <div className={style["info-span"]}>
+        <img src={image} alt="img" height="60px" width="260px" className={style["footer-img"]}/>
+        <p>Project Sakura is a fully free and Opensource Custom ROM built around LineageOS.</p>
+        <p><div style={{color:"#347ab7"}}><b>EMAIL ADDRESS </b></div>
+            hello@reallygreatsite.com</p>
```
Comment (ArunTeltia) [inline src/components/Footer/footer.js]: use @LordShenron email here shen.priyanshu@gmail.com or I am not sure If we want to give email, Lets just remove the email line

File: src/components/Footer/footer.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        <div className={style["footer-dash"]}>
+          <div className={style["footer-text"]}> 
+            <p>
+              <b style={{color:"white"}}>Made with <i class="fa fa-heart" style={{ color: "#DB3328" }}></i> by
+              Team Sakura</b>
+            </p>
+          </div>
+        </div>
+      </footer>
+    </React.Fragment>
```
Comment (ArunTeltia) [inline src/components/Footer/footer.js]: you can use <> instead of React.Fragment , just for future both do the same work they both are ghost tags

**Review by ArunTeltia — CHANGES_REQUESTED** [review]

**Comment by ArunTeltia** [comment]
Overall Really nice work @vedikaag99 , Add your name like in footer like in download.html 
at the right side If you want otherwise its okay by my side and Hope your exam goes well :)

File: src/components/Footer/footer.js (hunk 2 above)
Comment (vedikaag99) [reply src/components/Footer/footer.js]: done

**Review by ArunTeltia — APPROVED** [review]
Looks good to me, Merge @LordShenron

**Pull request merged**